app/session.py

```diff
@@ -38,27 +39,24 @@ def aiohttp_session_pr_331_rollback(self, identity, *, data, new,
 
 def setup(app_config):
     # Monkey patch aiohttp_session.py Session.__init__ method to remove PR 331 as above
-    Session.__init__ = aiohttp_session_pr_331_rollback
+    # Session.__init__ = aiohttp_session_pr_331_rollback
```

Comment:
Looks like we can delete this entirely now, including the function above that it was patching in. I checked the issue linked in the comment, and it's been resolved/closed for some time.